app/Http/View/Composers/CodeTrekApplicantsComposer.php

```diff
@@ -4,13 +4,15 @@
 
 use Illuminate\View\View;
 use Modules\CodeTrek\Entities\CodeTrekApplicant;
+use Modules\CodeTrek\Entities\CodeTrekFeedbackCategories;
 
 class CodeTrekApplicantsComposer
 {
     public function compose(View $view)
     {
         $codeTrekApplicants = CodeTrekApplicant::orderBy('first_name', 'asc')->get();
 
-        $view->with('codeTrekApplicants', $codeTrekApplicants);
+        $feedback_categories = CodeTrekFeedbackCategories::all();
+        $view->with('codeTrekApplicants', $codeTrekApplicants)->with('feedback_categories', $feedback_categories);
```

Comment:
Can we write like this?
Please test locally once, and let me know, it's works or not.
```suggestion
        $view->with([
    'codeTrekApplicants' => $codeTrekApplicants,
    'feedback_categories' => $feedback_categories,
]);
```

Reply:
Yes, it is working.